### PR TITLE
Provide overview of current implementations

### PR DESCRIPTION
**Pull request by tomkerkhove**
Provide overview of current implementations of Service Mesh Interface.

Relates to https://github.com/deislabs/smi-spec/issues/111

### CODE REVIEW

**Comment by tomkerkhove** [comment]
As an alternative we can add this remark but feels like overdoing it:
image

**Comment by grampelberg** [comment]
@tomkerkhove I'm good leaving out the note, but splitting into integrations/implementations like we did on the readme would be awesome! Thank you *so* much for getting this together!

**Comment by tomkerkhove** [comment]
@grampelberg Done!

**Comment by grampelberg** [comment]
Hmm, the logo seems to be messed up in the preview. Is that something you did or was it always broken?

```
https://deploy-preview-7--smi-spec.netlify.comimg/favicon.ico
```

Looks like hugo's not putting the `/` in there between the com and img.

**Comment by tomkerkhove** [comment]
This seems to be like this for other PRs as well and didn't change this but will have a look

**Comment by tomkerkhove** [comment]
It's defined in https://github.com/deislabs/smi-spec.io/blob/master/config.toml#L1 but not in https://github.com/deislabs/smi-spec.io/blob/master/netlify.toml#L1 which is causing this AFAIK, but not an expert.

**Comment by grampelberg** [comment]
@tomkerkhove good enough for me, I couldn't figure out how it'd be a regression but I wanted to double check.

**Comment by bridgetkromhout** [comment]
Looking at the Netlify preview, this is definitely wrong: https://deploy-preview-7--smi-spec.netlify.com/

image

**Comment by tomkerkhove** [comment]
That's right but not related to something I've changed. Anyhow, opened #8

**Comment by bridgetkromhout** [comment]
This is a little challenging to review since there are so many lines where the changes are just whitespace. I _think_ it's fine, but I can't entirely rule out the possibility of problematic changes I haven't noticed.

**Comment by tomkerkhove** [comment]
Ah dammit, I've messed it up with the rebase :/

It's mainly consolidating the "partnerships" into "ecosystem" which now also contains "integrations" & "implementations". https://github.com/deislabs/smi-spec.io/pull/7/files#diff-ca4afbbbc9dfb4e3354db103e5a14757R80

Sorry about that.

**Comment by tomkerkhove** [comment]
This:

**Comment by bridgetkromhout** [comment]
Thanks, @tomkerkhove. Comparing the live site and the preview, visually, this looks good to me. If another maintainer like @lachie83 or @michelleN have a chance to look over the logic and agree with the integrations/implementations split, then I think this is good to go. Thanks!

**Comment by stefanprodan** [comment]
Can we please use this logo for Flagger https://github.com/weaveworks/flagger/blob/master/docs/logo/flagger-weaveworks.svg

**Comment by tomkerkhove** [comment]
Certainly!

**Comment by lachie83** [comment]
Let's add this to the agenda for the next community meeting. I'm not sure whether or not we need explicit approval to use these logos.

**Comment by bridgetkromhout** [comment]
> Let's add this to the agenda for the next community meeting. I'm not sure whether or not we need explicit approval to use these logos.

I have added this to the agenda for March 18th.

**Comment by tomkerkhove** [comment]
I'll try to join the standup as well but would certainly help discoverability!

File: themes/smi/layouts/index.html [hunk 1]
```diff
@@ -1,129 +1,156 @@
...
-            <!-- <h4>Slack Channel</h4> -->
-            <div class="panel-wrapper small-12 large-8">
-              <p>Sign up for the <a href="http://smi-spec.slack.com/"><strong>SMI Slack</strong></a> for related discussion.</p>
+  </div>
+</div>
+
+<div class="row row-full" id="credits">
+  <h2 class="text-center sep sep-sides">Ecosystem</h2>
+  <div class="small-12 large-10 large-centered columns">
+    <h3 class="text-center sep sep-sides" id="implementations">Implementations</h3>
```
Comment (leecalcote) [inline themes/smi/layouts/index.html]: @tomkerkhove we'll want to ensure Meshery's inclusion here. Please see https://layer5.io/brand for a logo pack.

Comment (tomkerkhove) [reply themes/smi/layouts/index.html]: Added in the next PR push, but noticed that "Download Logo Asset" buttons are broken on https://layer5.io/brand

**Comment by tomkerkhove** [comment]
@grampelberg @bridgetkromhout @lachie83 I've removed the seperation to align with https://github.com/deislabs/smi-spec/pull/114
image

@leecalcote Meshery is now listed!

**Comment by bridgetkromhout** [comment]
We discussed this at today's meeting; we're looking to get approval for logo use as a comment in this PR from @ilevine @grampelberg @nicholasjackson (already have @stefanprodan).

**Comment by bridgetkromhout** [comment]
@tomkerkhove please sign your commits in this PR! Thanks.

**Comment by tomkerkhove** [comment]
Good to go now and happy to sit and wait for all the sign-offs.

Let me know where I can help.

**Comment by bridgetkromhout** [comment]
Hi @tomkerkhove - please take a look at these merge conflicts (which cropped up because we're making some updates around the theme) - thanks!

**Comment by tomkerkhove** [comment]
Done!

**Comment by lucperkins** [comment]
@tomkerkhove Dang, you are not messing around! 😄

**Comment by tomkerkhove** [comment]
If it takes less than 5 minutes, just do it now 😅

**Comment by lucperkins** [comment]
@tomkerkhove Words to live by. Once this gets merged I'll merge #11 and then do some templating changes that'll make it easier to modify these logo lists and such.

**Comment by bridgetkromhout** [comment]
Great work keeping up, @tomkerkhove!  I think we could merge this once the not-yet-approved images are not longer in the PR (maybe just replaced by words). 

Could you rebase off master so this PR isn't full of commits which were already merged into master? That makes seeing what changed so much easier. Thanks!

**Comment by tomkerkhove** [comment]
I will - What is the list of approved logos? (just to get things correct)

**Comment by bridgetkromhout** [comment]
I did just cause a merge conflict here (sorry) as well.

> I will - What is the list of approved logos? (just to get things correct)

I think this PR is where people were going to answer around the logos (https://github.com/servicemeshinterface/smi-spec.io/pull/7#issuecomment-600918237). If we get it merged we could open an issue calling out to them again.

**Comment by bridgetkromhout** [comment]
If you want to get this PR merged, maybe just list them all except Flagger as text for now and then  we can iteratively get logos added as they're approved?

**Comment by tomkerkhove** [comment]
Should be good now, added Linkerd as well based on #20

**Comment by bridgetkromhout** [comment]
I'll merge this now so we can move the website updates forward. We'll keep adding logos as they're approved.

**Pull request merged**